front/lib/api/assistant/conversation.ts

```diff
@@ -427,7 +461,7 @@ export async function getConversation(
   });
 
   // We need to escape the type system here to create content. We pre-create an array that will hold
```

Comment:
I'm quite annoyed by this... but don't have a better (reasonable) solution

Reply:
cc @spolu in case you have an idea

Reply:
I don't. I wrote that comment :)